lib/runner/extensions/item.command.js

```diff
@@ -163,6 +163,14 @@ module.exports = {
                         });
                     }
 
+                    if (shouldSkipExecution) {
+                        this.triggers.item(prereqExecutionError, coords, item);
+
+                        return callback && callback.call(this, null, {
```

Comment:
Let's pass prereqExecutionError instead of null for sanity.

Reply:
Done